### PR TITLE
Create Sensor Object class that outlines the attributes of a sensor object and 3 main functions that can be used on a sensor object

### LINKED ISSUE #76: Create Sensor Object class that outlines the attributes of a sensor object and 3 main functions that can be used on a sensor object

### PR DESCRIPTION
**Pull request by Birdbh**
Fixes #76

### CODE REVIEW

File: src/sensors/sensor.py [hunk 1]
```diff
@@ -1,11 +1,51 @@
...
+    def __init__(self, id, name, description, type, value):
         self.id = id
         self.name = name
         self.description = description
         self.type = type
         self.value = value
+        self.forcasted_values = {}
 
     def __str__(self):
-        return f"Sensor: ID:{self.id}, NAME:{self.name}, DESCRIPTION:{self.description}, DATA-TYPE:{self.type}, {self.value}"
```
Comment (JamesShadowfyre) [inline src/sensors/sensor.py:11]: Please confirm this

File: src/sensors/sensor.py [hunk 2]
```diff
@@ -1,11 +1,51 @@
-
 class Sensor:
-    def __init__(self, id, name, description, type, location, value):
```
Comment (JamesShadowfyre) [inline src/sensors/sensor.py:3]: Please confirm this

File: src/__pycache__/__init__.cpython-39.pyc
Comment (JamesShadowfyre) [inline src/__pycache__/__init__.cpython-39.pyc:1]: add .gitignore to pycache

**Review by JamesShadowfyre — COMMENTED** [review]
Branch adjusted to fix conflict error. Please confirm the right versions of the two function headers were merged in.  Add a .gitignore to your pycache file, as it is not needed in the repo.

File: src/sensors/sensor.py [hunk 3]
```diff
@@ -1,11 +1,51 @@
...
+    def get_type(self):
+        return self.type
+
+    def set_value(self, new_value):
+        self.value = new_value
+
+    def get_value(self):
+        return self.value
+
+    def set_forecast_values(self, forcasted_values):
```
Comment (Jcp149) [inline src/sensors/sensor.py:43]: will you ever need to alter the forcast values? if so, this could be difficult!

**Review by Birdbh — COMMENTED** [review]
>  

I dont undertand what you mean, can you explain more?

**Comment by JamesShadowfyre** [comment]
which comment are you responding to?

**Review by Birdbh — COMMENTED** [review]
> will you ever need to alter the forcast values? if so, this could be difficult! 

I dont undertand what you mean, can you explain more?

File: src/sensors/sensor.py (hunk 2 above)
Comment (Birdbh) [reply src/sensors/sensor.py:3]: Dendied, not confirm

**Comment by Jcp149** [comment]
like will we ever have to do more than just get and set the forecast dictionaries? like if we have to append values, that could be difficult but if not it all looks good to me

**Pull request merged**